src/operator/OrgOverlay.tsx

```diff
@@ -108,7 +108,7 @@ const OrgOverlay: FC = () => {
                       text="View Usage Dashboard"
                       target={LinkTarget.Blank}
                       className="overlay-button--link"
-                      href={`https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/0649b03029c49000?vars%5Borgid%5D=${orgID}`}
+                      href={`https://us-east-1-2.aws.cloud2.influxdata.com/orgs/5d59ccc5163fc318/dashboards/0988da0fd78a7003?vars%5Borgid%5D=${orgID}`}
```

Comment:
I remember talking in the Friday standup on how the previous tools was hard coded here. Do we want to change the approach?

Otherwise looks good!

Reply:
@ChitlangeSahas Correct!  But per AIM, they want us to keep this hardcoded.

Reply:
Got it 👍

Reply:
doing a code search of `influxdb.aws.influxdata.io` reveals a couple of places in `flows` where this url is hardcoded. I think it might be a good idea to store `https://us-east-1-2.aws.cloud2.influxdata.com` into a variable called `TOOLS_URL`, which is still keeping it hardcoded (rather than based on a config value), but makes it slightly easier to work with.